.github/workflows/release-new-action-version.md

```diff
@@ -0,0 +1,27 @@
+name: Release new action version
```

Comment:
Hi ! Shouldn't it be `release-new-action-version.yml` instead of `release-new-action-version.md`

Reply:
Fixed! Thanks!